lib/ui/platform_dispatcher.dart

```diff
@@ -1289,8 +1289,18 @@ class PlatformConfiguration {
 /// An immutable view configuration.
 class ViewConfiguration {
   /// A const constructor for an immutable [ViewConfiguration].
+  ///
+  /// When constructing a view configuration, supply either the `view` or the
+  /// `window` property, but not both since the `view` and `window` property
+  /// are backed by the same instance variable.
```

Comment:
nit: since window and view are actual properties of this class put them in `[]` instead of `. That way, dartdoc directly links to them.